unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Definitions/Animator.proto

```diff
@@ -10,9 +10,9 @@ message PBAnimator {
 message PBAnimationState {
   string name = 1;
   string clip = 2;
-  bool playing = 3;
-  float weight = 4;
-  float speed = 5;
-  bool loop = 6;
-  bool should_reset = 7;
+  optional bool playing = 3;
+  optional float weight = 4; // default=1.0s
+  optional float speed = 5; // default=1.0
+  optional bool loop = 6; // default=true
+  optional bool should_reset = 7;
```

Comment:
playing and should_reset doesn't have a default value specified

Reply:
It's correct. The default value is `false` and is not specified because is the `default value` that C# assigned.

We need to change this. But we're going to do it when we generate the code automatically.